src/containers/App/App.js

```diff
@@ -104,26 +115,65 @@ export /* istanbul ignore next */ class App extends Component {
     if (webSocketConnected && prevWebSocketConnected === false) {
       this.fetchData();
     }
+    if (this.props.tenantNamespace) {
+      this.props.selectNamespace(this.props.tenantNamespace);
+    }
   }
 
   componentWillUnmount() {
     this.props.onUnload();
   }
 
-  fetchData() {
-    this.props.fetchExtensions();
-    this.props.fetchInstallProperties();
-    this.props.fetchNamespaces();
+  async fetchData() {
+    try {
+      const properties = await this.props.fetchInstallProperties();
```

Comment:
I'm not happy with this, directly using the result of the action looks bad isn't it ?

Reply:
I guess the action should dispatch to the store and the container should in turn react to store change...

Reply:
Changed it, should be better hopefully.